### DISCUSSION TITLE
Mace-jax Molecular Dynamics

### DISCUSSION THREAD

**Discussion by owen-rett** [ACEsuit/mace#175] [Q&A] [answered]
Hello,
Noticed on the README.md, that the mace-jax version of mace is up to 2x faster in evaluation. Is mace-jax compatible with any MD packages, eg. the jax-md package? or is it necessary to use it with ase's molecular dynamics?

**Comment by ilyes319** [comment, marked as answer]
We have a very untested jax md calculator with mace-jax. There really has not been any test, so it surely has some bugs. If you are willing to test that, than yes there is one.
